Formula/provengo.rb

```diff
@@ -0,0 +1,22 @@
+class Provengo < Formula
+  desc "Model-based testing and automation system"
+  homepage "https://provengo.tech/"
+  url "https://downloads.provengo.tech/installer/provengo-0.7.5.zip"
+  sha256 "f632d7c59ef0e2aaddb223c59906bd776acf30450167e384db486d48c99f5f5b"
+  license :cannot_represent
+
+  depends_on "graphviz"
+  depends_on "openjdk@11"
```

Comment:
you might need to override the JAVA_HOME for this?

Also any reasons why we cannot use openjdk?